.eslintrc.json

```
@@ -1,12 +1,26 @@
 {
-    "parser": "babel-eslint",
-    "extends": ["airbnb-base", "prettier"],
-    "plugins": ["import"],
+    "parser": "@typescript-eslint/parser",
+    "extends": [
+        "plugin:@typescript-eslint/recommended",
+        "prettier"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 2019,
+        "sourceType": "module"
+    },
     "rules": {
-        "no-prototype-builtins": "off",
         "indent": [
             "error",
             4
-        ]
+        ],
+        "semi": [
+            "error",
+            "always"
+        ],
+        "quotes": [
+            "error",
+            "single"
```

```suggestion
            "backtick"
```
This is your choice but the reason I prefer backticks is explained really well here
https://ponyfoo.com/articles/template-literals-strictly-better-strings

---

This will not be changed due to performance. 

The following test was performed on [jsperf](jsperf.com) with Firefox 70 and Windows 10:
```
string concatenation    1,380,408,753    ±5.09%
es6                     1,354,202,670    ±5.01%
```

```js
Benchmark.prototype.setup = function() {
    var precalculatedIso = (new Date()).toISOString();
};

// string concatenation
var test = "<body>"+
  "<article>"+
    "<time datetime='" + precalculatedIso +"'>"+ precalculatedIso +"</time>"+
  "</article>"+
"</body>";

// ES6
var test = `<body>
  <article>
    <time datetime='${ precalculatedIso }'>${ precalculatedIso }</time>
  </article>
</body>`;
```

---

TLDR of the article: The only time backticks are slower is when u need to add variables to it. However, when you want to use variables devs will prefer backticks anyway.

**Backticks are faster here**
```js
const testSingle = 'anything'
const testDouble = "anything"
const testBackticks = `anything`
```

vs

**Backticks are slower here**
```js
const testSingle = "anything" + var
const testDouble = "anything" + double
const testBackticks = `anything ${double}`
```


Full article: https://medium.com/javascript-in-plain-english/are-backticks-slower-than-other-strings-in-javascript-ce4abf9b9fa